boards/pba-d-01-kw2x/Makefile.dep

```diff
@@ -0,0 +1,4 @@
+ifneq (,$(filter ng_netif,$(USEMODULE)))
+  USEMODULE += kw2xrf
```

Comment:
sry, I don´t got the point correctly. What would be the alternative? You mean ```  USEMODULE += ng_ieee802154``` is missing here?